samples/client-sdk/connectivity-tests/src/call/controls.js

```diff
@@ -0,0 +1,76 @@
+import { hideTestResults, removeAllParticipantEles } from './dom.js';
+
+/**
+ * Configures the test button onclick handler
+ * @param {()} handler
+ */
+export function setupTestBtn(handler) {
+  const testBtn = getTestBtn();
+  testBtn.onclick = (ev) => {
+    ev.preventDefault();
+    handler();
+  };
+}
+
+/**
+ * Enables the test button
+ */
+export function enableTestBtn() {
+  const testBtn = getTestBtn();
+  testBtn.disabled = false;
+}
+
+/**
+ * Disables the test button
+ */
+export function disableTestBtn() {
```

Comment:
I know there's an established pattern here but one line (`document.getElementById('test').disabled = true`) would replace two functions and their usage (disableTestBtn and getTestBtn). I guess my general feedback is that refactoring all these into reusable functions makes it harder to read -- IMO at least!

Reply:
I think since the test button retrieval is reused three times, rewriting the retrieval logic each time would make it more error prone (to typos as well as if the button ID ever changed, for example). I _could_ turn this into a single `updateTestBtn()` function or something instead of having `enable` and `disable` funds, but that would still leave two usages and imo would possibly make the usage less intuitive (i.e., whoever is reading the invocation location will clearly see what is about to happen by the function name, whereas with an "update" name they may need to read what the function does to confirm what's about to happen.

_But_ what I could do is still do this all on one line: `getTestBtn().disabled = true`!

Reply:
Fair, your call!